### PR TITLE
feat: Save TransferType and AuthorEmail

### PR DESCRIPTION
**Pull request by valentinperignon**

### CODE REVIEW

File: STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.multiplatform_swisstransfer.common.models
+
+enum class TransferType(val value: String) {
+    LINK("link"),
+    QR_CODE("qrcode"),
+    PROXIMITY("proximity"),
+    MAIL("mail")
+}
```
Comment (KevinBoulongne) [inline STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt:25]: We prefer to end each file with an empty line.

Comment (FabianDevel) [inline STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt]: Add a blank line

File: STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.multiplatform_swisstransfer.common.models
+
+enum class TransferType(val value: String) {
+    LINK("link"),
+    QR_CODE("qrcode"),
+    PROXIMITY("proximity"),
+    MAIL("mail")
```
Comment (FabianDevel) [inline STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt]:
```suggestion
    MAIL("mail"),
```

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.multiplatform_swisstransfer.common.models
+
+enum class TransferType {
+    LINK,
+    QR_CODE,
+    PROXIMITY,
+    MAIL
```
Comment (KevinBoulongne) [inline STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt]:
```suggestion
    MAIL,
```

**Review by FabianDevel — CHANGES_REQUESTED** [review]

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt (hunk 2 above)
Comment (FabianDevel) [reply STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt]: too late 🥈

File: STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt (hunk 1 above)
Comment (FabianDevel) [reply STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt]: too late 🥈

**Review by KevinBoulongne — APPROVED** [review]

**Review by FabianDevel — APPROVED** [review]

**Pull request merged**